ansible/roles/fanout/templates/arista_7260_connect.j2

```diff
@@ -1,5 +1,8 @@
 !
 config
+{% if deploy_leaf %}
+vlan {{ dev_vlans | list | join(',') }}
```

Comment:
Can this generate a very long config line like:
vlan 2000,2001,2002,2003,...

Reply:
It's possible in theory. But the ```dev_vlans``` is a sorted and grouped list, which means all consecutive values will be merged in the same range. So I think it won't cause trouble.
But I have another concern here, that is we keep adding vlan_range vlan_list, but never clear the unused ones. Luckily, it probably cause no trouble because we have a vlan list for each trunk.